code/__HELPERS/files.dm

```diff
@@ -15,6 +15,13 @@
 /proc/get_maps(root="maps/voting/")
 	var/list/maps = list()
 	var/recursion_limit = 20 //lots of maps waiting to be played, feels like TF2
+	//Check if we can add the special maps
+	var/Lamprey = root + "Lamprey/vgstation13.dmb"
+	if(Lamprey && (score["crewscore"] <= -15000))
+		maps += Lamprey
```

Comment:
you can't just add this to the end of maps, it needs to be associated with its path